app/assets/javascripts/global.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function() {
+    var copyTextBtn = document.querySelector('.copy-text-btn');
```

Comment:
Why did you write all of this in pure Javascript and not use JQuery?

Reply:
Almost entirely because I'm not familiar with JQuery.

Reply:
oh no